includes/rest/class-following-controller.php

```diff
@@ -69,49 +95,31 @@ public static function get( $request ) {
 		 */
 		\do_action( 'activitypub_rest_following_pre' );
 
-		$json = new \stdClass();
-
-		$json->{'@context'} = \Activitypub\get_context();
-
-		// phpcs:disable WordPress.NamingConventions.ValidVariableName.UsedPropertyNotSnakeCase
-		$json->id        = get_rest_url_by_path( sprintf( 'actors/%d/following', $user->get__id() ) );
-		$json->generator = 'http://wordpress.org/?v=' . get_masked_wp_version();
-		$json->actor     = $user->get_id();
-		$json->type      = 'OrderedCollectionPage';
-		$json->partOf    = get_rest_url_by_path( sprintf( 'actors/%d/following', $user->get__id() ) );
+		$response = array(
+			'@context'  => \Activitypub\get_context(),
+			'id'        => get_rest_url_by_path( \sprintf( 'actors/%d/following', $user->get__id() ) ),
+			'generator' => 'https://wordpress.org/?v=' . get_masked_wp_version(),
```

Comment:
@pfefferle Is switching from `http` to `https` going to be a problem?

Reply:
on the wordpress.org URL? no, should be no problem.